### ISSUE TITLE
crash with google_compute_instance without boot_disk/initialize_params or source

### ISSUE THREAD

**Issue by matti** [terraform-providers/terraform-provider-google#578]
```
panic: runtime error: invalid memory address or nil pointer dereference
2017-10-12T11:18:29.719+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: [signal SIGSEGV: segmentation violation code=0x1 addr=0x140 pc=0x1c2e5c4]
2017-10-12T11:18:29.719+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4:
2017-10-12T11:18:29.719+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: goroutine 133 [running]:
2017-10-12T11:18:29.719+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: github.com/terraform-providers/terraform-provider-google/google.(*ComputeOperationWaiter).RefreshFunc.func1(0xc42004df20, 0x1, 0x0, 0x0, 0x0, 0x0)
2017-10-12T11:18:29.719+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform-providers/terraform-provider-google/google/compute_operation.go:27 +0x64
2017-10-12T11:18:29.719+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/resource.(*StateChangeConf).WaitForState.func1(0xc4203e8000, 0xc4201dc2a0, 0xc420132340, 0xc4203220c0, 0xc4202722c8, 0xc4202722c0)
2017-10-12T11:18:29.719+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/resource/state.go:103 +0x4e0
2017-10-12T11:18:29.719+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: created by github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/resource.(*StateChangeConf).WaitForState
2017-10-12T11:18:29.719+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/resource/state.go:80 +0x1d0
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: panic: interface conversion: interface {} is nil, not *compute.Operation
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4:
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: goroutine 131 [running]:
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: github.com/terraform-providers/terraform-provider-google/google.computeOperationWaitTime(0xc4204b8240, 0x0, 0xc42071a4a8, 0x8, 0x2025749, 0x12, 0x4, 0x1b19720, 0xc4202df2d0)
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform-providers/terraform-provider-google/google/compute_operation.go:88 +0x332
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: github.com/terraform-providers/terraform-provider-google/google.computeSharedOperationWaitTime(0xc4204b8240, 0x1ddfc40, 0x0, 0xc42071a4a8, 0x8, 0x4, 0x2025749, 0x12, 0x0, 0x0)
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform-providers/terraform-provider-google/google/compute_shared_operation.go:19 +0xa3
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: github.com/terraform-providers/terraform-provider-google/google.resourceComputeInstanceCreate(0xc4202df2d0, 0x1dcfb60, 0xc4204b8240, 0x24, 0x288b6a0)
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform-providers/terraform-provider-google/google/resource_compute_instance.go:793 +0x2a8b
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/schema.(*Resource).Apply(0xc420067260, 0xc420700370, 0xc4205e17e0, 0x1dcfb60, 0xc4204b8240, 0x1, 0xc42057b6b0, 0xc42057b6b0)
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/schema/resource.go:193 +0x3b6
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/schema.(*Provider).Apply(0xc4201dd1f0, 0xc420700320, 0xc420700370, 0xc4205e17e0, 0x2bf7d90, 0x0, 0x18)
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/helper/schema/provider.go:259 +0xa4
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/plugin.(*ResourceProviderServer).Apply(0xc42050a0c0, 0xc4205e1380, 0xc420594910, 0x0, 0x0)
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/opt/teamcity-agent/work/222ea50a1b4f75f4/src/github.com/terraform-providers/terraform-provider-google/vendor/github.com/hashicorp/terraform/plugin/resource_provider.go:488 +0x57
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: reflect.Value.call(0xc420378120, 0xc4203f2190, 0x13, 0x2017ccf, 0x4, 0xc42004df20, 0x3, 0x3, 0xc4202e2ee8, 0xc42036a028, ...)
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/usr/local/go/src/reflect/value.go:434 +0x906
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: reflect.Value.Call(0xc420378120, 0xc4203f2190, 0x13, 0xc4202e2f20, 0x3, 0x3, 0xc400000005, 0x0, 0x0)
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/usr/local/go/src/reflect/value.go:302 +0xa4
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: net/rpc.(*service).call(0xc420132380, 0xc42014a780, 0xc42071a2f8, 0xc4205e8080, 0xc420720dc0, 0x1cd34c0, 0xc4205e1380, 0x16, 0x1cd3500, 0xc420594910, ...)
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/usr/local/go/src/net/rpc/server.go:381 +0x142
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: created by net/rpc.(*Server).ServeCodec
2017-10-12T11:18:29.728+0300 [DEBUG] plugin.terraform-provider-google_v1.0.1_x4: 	/usr/local/go/src/net/rpc/server.go:475 +0x36b
2017/10/12 11:18:29 [TRACE] root: eval: *terraform.EvalWriteState
2017/10/12 11:18:29 [TRACE] root: eval: *terraform.EvalApplyProvisioners
2017/10/12 11:18:29 [TRACE] root: eval: *terraform.EvalIf
2017/10/12 11:18:29 [TRACE] root: eval: *terraform.EvalWriteState
2017/10/12 11:18:29 [TRACE] root: eval: *terraform.EvalWriteDiff
2017/10/12 11:18:29 [TRACE] root: eval: *terraform.EvalApplyPost
2017/10/12 11:18:29 [ERROR] root: eval: *terraform.EvalApplyPost, err: 1 error(s) occurred:

* google_compute_instance.instance: unexpected EOF
2017/10/12 11:18:29 [ERROR] root: eval: *terraform.EvalSequence, err: 1 error(s) occurred:

* google_compute_instance.instance: unexpected EOF
2017/10/12 11:18:29 [TRACE] [walkApply] Exiting eval tree: google_compute_instance.instance
2017/10/12 11:18:29 [TRACE] dag/walk: upstream errored, not walking "provider.google (close)"
2017/10/12 11:18:29 [TRACE] dag/walk: upstream errored, not walking "output.network_interfaces"
2017/10/12 11:18:29 [TRACE] dag/walk: upstream errored, not walking "meta.count-boundary (count boundary fixup)"
2017/10/12 11:18:29 [TRACE] dag/walk: upstream errored, not walking "root"
2017-10-12T11:18:29.732+0300 [DEBUG] plugin: plugin process exited: path=/Users/mpa/dev/ag/opsgyver/terraform/google/.terraform/plugins/darwin_amd64/terraform-provider-google_v1.0.1_x4
2017/10/12 11:18:29 [TRACE] Preserving existing state lineage "8526ace3-d590-4bc4-a1c3-57fdd51b9488"
2017/10/12 11:18:29 [TRACE] Preserving existing state lineage "8526ace3-d590-4bc4-a1c3-57fdd51b9488"
2017/10/12 11:18:29 [TRACE] Preserving existing state lineage "8526ace3-d590-4bc4-a1c3-57fdd51b9488"
2017/10/12 11:18:29 [TRACE] Preserving existing state lineage "8526ace3-d590-4bc4-a1c3-57fdd51b9488"
2017/10/12 11:18:29 [DEBUG] plugin: waiting for all plugin processes to complete...
2017-10-12T11:18:29.734+0300 [WARN ] plugin: error closing client during Kill: err="connection is shut down"



!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Terraform crashed! This is always indicative of a bug within Terraform.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Terraform[1] so that we can fix this.

When reporting bugs, please include your terraform version. That
information is available on the first line of crash.log. You can also
get it by running 'terraform --version' on the command line.

[1]: https://github.com/hashicorp/terraform/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

**Comment by rosbo** [comment]
Duplicate of #553. Fixed in #556. 

I will keep it open for now until our next release to avoid having people filling the same issue over and over :)

**Comment by rosbo** [comment]
We released a new version (1.1.0) of the google cloud provider yesterday. Please reopen this issue if the problem persists. 

Thank you
